WordClock/Config.h

```diff
@@ -0,0 +1,24 @@
+#ifndef Config_h
+#define Config_h
+
+#define CONFIG_VERSION "001"
```

Comment:
Zou 4 chars moeten zijn gezien de array

Reply:
Is meer een gereserveerd aantal karakters. 
Een SSID is ook niet altijd de max length die er voor gereserveerd is.